commit f9d7f383212961f1d9dd0466ca470a78c7fce980 (refs/changes/01/1701/1)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2017-03-22T17:48:18+00:00 (2 years, 7 months ago)
    
    NAPI-401 VL2 and VL3 events should be inserted with null etags

diff --git a/lib/consumer.js b/lib/consumer.js
index f135a09..27f0430 100644
--- a/lib/consumer.js
+++ b/lib/consumer.js
@@ -312,8 +312,7 @@ function logReq(opts, callback) {
      * limit as the _maximum bytes_ to return; even should varpd requests bytes
      * that would total >1000 messages, we can safely return the first 1000.
      */
-    var req = opts.moray.findObjects(NAMES.net_events, filter, searchOpts,
-        true);
+    var req = opts.moray.findObjects(NAMES.net_events, filter, searchOpts);
 
     req.once('error', function searchError(err) {
         opts.log.error(err, 'logreq search error');
diff --git a/lib/producer.js b/lib/producer.js
index 4fd9818..f59f406 100644
--- a/lib/producer.js
+++ b/lib/producer.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright 2016, Joyent, Inc.
+ * Copyright 2017, Joyent, Inc.
  */
 
 /*
@@ -89,10 +89,7 @@ function overlayMappingBatch(opts) {
         bucket: NAMES.mac_ip,
         key: rec.key,
         operation: 'put',
-        value: rec.value,
-        options: {
-            // XXX: set etag?
-        }
+        value: rec.value
     };
 }
 
@@ -186,9 +183,6 @@ function underlayMappingBatch(opts) {
             cn_uuid: opts.cn_uuid,
             ip: opts.ip,
             port: opts.port
-        },
-        options: {
-            // XXX: set etag?
         }
     };
 }
@@ -270,6 +264,9 @@ function vl2CnEventBatch(opts) {
             bucket: NAMES.net_events,
             key: uuid,
             operation: 'put',
+            options: {
+                etag: null
+            },
             value: {
                 cn_uuid: cn,
                 vnet_id: opts.vnet_id,
@@ -308,6 +305,9 @@ function vl3CnEventBatch(opts) {
             bucket: NAMES.net_events,
             key: uuid,
             operation: 'put',
+            options: {
+                etag: null
+            },
             value: {
                 cn_uuid: cn,
                 vnet_id: opts.vnet_id,
